pastasoss-environment-3.9.yml

```diff
@@ -9,4 +9,4 @@ dependencies:
   - matplotlib=3.6.2
   - pytest=7.2.0
   - scipy=1.10.1
-  
+  - astropy
```

Comment:
Since we are no longer supporting 3.9 in our testing matrix, we should remove the environment too!

Reply:
Environment has been removed.